### ISSUE TITLE
Feature: support batch with "select for update"

### ISSUE THREAD

**Issue by pengten** [seata/seata#5437]
## Why you need it?
 Is your feature request related to a problem? Please describe in details
support like this:
....for update
....for update

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

**Comment by slievrly** [comment]
I think this is the multi-table scenario. Are they coming from real end-user needs?

**Comment by pengten** [comment]
> I think this is the multi-table scenario. Are they coming from real end-user needs?

yes, this question comes from a real project scenario within the company.
